<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MotionTreeEditorGui</class>
 <widget class="QWidget" name="MotionTreeEditorGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Track</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="m_buttonTrack1">
      <property name="text">
       <string>1</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="m_buttonTrack2">
      <property name="text">
       <string>2</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="m_buttonTrack3">
      <property name="text">
       <string>3</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="m_buttonTrack4">
      <property name="text">
       <string>4</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="m_buttonTrack5">
      <property name="text">
       <string>5</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="m_buttonTrack6">
      <property name="text">
       <string>6</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="m_buttonTrack7">
      <property name="text">
       <string>7</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="m_buttonTrack8">
      <property name="text">
       <string>8</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Tree default action</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="m_le_defaultAction"/>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>20</y>
     <width>211</width>
     <height>121</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>action</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>variable</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
